route/route.go

```diff
@@ -120,6 +129,10 @@ func (r *Router) panic(w http.ResponseWriter, req *http.Request) {
 	panic("panic? never!")
 }
 
+func (r *Router) version(w http.ResponseWriter, req *http.Request) {
+	w.Write([]byte(fmt.Sprintf(`{"source":"samproxy","version":%s}`, r.versionStr)))
```

Comment:
is `versionStr` an actual version string? Or just an id? Should it be wrapped in quotes here so it's returned as a string?

Reply:
yes! it should! thank you.